test/e2e/util.go

```diff
@@ -201,6 +218,10 @@ func RunJobListener(t *testing.T, clients *test.Clients) (string, context.Cancel
 		if err != nil {
 			t.Errorf("Error cleaning up Job %s", job.Name)
 		}
+		err = clients.KubeClient.CoreV1().Pods(job.Namespace).DeleteCollection(context.Background(), metav1.DeleteOptions{}, metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", jobNameKey, name)})
+		if err != nil {
+			t.Errorf("Error cleaning up pod for Job %s", job.Name)
```

Comment:
nit: since it's collection (same for other collection)

```suggestion
			t.Errorf("Error cleaning up pods for Job %s", job.Name)
```